suggestions.md

```diff
@@ -0,0 +1,228 @@
+# Suggestions Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/suggestions**   
+
+_Unsupported._ The suggestions can be retrieved only by source and target or via direct link, see the single entry and search method below. 
+
+### single entry
+**GET api/integration/suggestions/<:suggestion-id>**
+
+It returns the data from one suggestion. The suggestion-id is a String generated by the source, as it must be unique for source, target and identified object to be imported it is usually represented by a combination of such elements
+
+sample for a suggestion /api/integration/suggestions/reciter:gf3d657-9d6d-4a87-b905-fef0f8cae26:24694772
+```json
+{
+    "id": "reciter:gf3d657-9d6d-4a87-b905-fef0f8cae26:24694772",
+    "display": "publication one",
+    "source": "reciter",
+    "score": "62.7",
+    "external-source-uri": "https://dspace7.4science.cloud/server/api/integration/reciterSourcesEntry/pubmed/entryValues/24694772",
+    "evidences": {
+      "acceptedRejectedEvidence": {
```

Comment:
see [reply](https://github.com/DSpace/RestContract/pull/192#discussion_r1490900381).

Reply:
As I noted in my [later reply](https://github.com/DSpace/RestContract/pull/192/files#r1491149360), I think we need to keep the list of "evidences" **realistic**, otherwise developers will be confused if they cannot find these in DSpace.  

I'm OK with providing **one example** of a custom evidence.  But, currently this REST Contract PR has a list of 9 evidences _all of which do not exist in DSpace_.  I propose we update this to say something like:
```
        "AuthorNamesScorer" : {
          "score" : "100.00",
          "notes" : "some notes, empty or null"
        },
        "DateScorer" : {
          "score" : "0.00",
          "notes" : "some notes, empty or null"
        },
        "customEvidence": {
          "score" : "23.7",
          "notes" : "some notes, empty or null"
        },
```

Reply:
@frabacche : Could you update the list of evidences in this section of `suggestions.md` the same way that you updated them below?  This section still has the list of invalid "evidences"